angular-workspace/projects/ni/nimble-angular/src/directives/combobox/tests/nimble-combobox-control-value-accessor.directive.spec.ts

```diff
@@ -29,8 +29,12 @@ describe('Nimble combobox control value accessor', () => {
         @Component({
             template: `
                 <nimble-combobox #combobox [(ngModel)]="selectedOption" (ngModelChange)="onModelValueChange($event)" [compareWith]="compareWith" [disabled]="selectDisabled" autocomplete="none">
-                    <nimble-list-option *ngFor="let option of selectOptions" [ngValue]="option">{{ option?.name ?? nullValueString }}</nimble-list-option>
-                    <nimble-list-option [ngValue]="dynamicOption">{{ dynamicOption?.name }}</nimble-list-option>
+                    <nimble-list-option *ngFor="let option of selectOptions" [ngValue]="option">
+                        {{ option?.name ?? nullValueString }}
```

Comment:
Can you add a comment saying that the test intentionally is leaving whitespace around the value. Or maybe a specific separate test where it more explicitly initializes the content of an attribute with extra white space and specifically tests that in the test name.

Main concern is I can easily see a refactoring changing the layout of this fixture and losing coverage of this test case.

Reply:
I've reverted this change, and now made one of the options have whitespace around the main text which both failed some previous tests as well as a new explicit test I've added.